src/Ironclad/Sdk/AuthCookieMiddleware.cs

```diff
@@ -56,5 +66,34 @@ await handler.HandleRequestAsync())
 
             await this.next.Invoke(context);
         }
+
+        private bool UserUsesLatestIos(HttpContext context)
+        {
+            var userAgent = context.Request.Headers["User-Agent"].ToString();
+            var groups = regex.Matches(userAgent);
+
+            if (groups.Any())
+            {
+                var captures = groups[0].Captures;
+                if (captures.Any())
+                {
+                    // Captured version might be in a form of a semver, ie. 'OS 10_3_0'
+                    var version = captures[0].Value
+                        .Replace("OS ", string.Empty, StringComparison.InvariantCultureIgnoreCase)
+                        .Split('_', StringSplitOptions.RemoveEmptyEntries)
+                        .First();
+
+                    if (int.TryParse(version, NumberStyles.Any, CultureInfo.InvariantCulture, out var v))
```

Comment:
We should be able to combine and inline these:
```suggestion
                    return int.TryParse(version, NumberStyles.Any, CultureInfo.InvariantCulture, out var v) && v >= 12;
```

Reply:
done